docs/source/release/v6.2.0/diffraction.rst

```diff
@@ -13,6 +13,7 @@ Powder Diffraction
 ------------------
 New features
 ############
+- New algorithm :ref:`CombineDiffCal <algm-CombineDiffCal>` to calibrate groups of pixels after cross correlation so that diffraction peaks can be adjusted to the correct positions
```

Comment:
You are missing the docs page for this algorithm so this will just fail to create the link, you need to create one at `docs/source/algorithms/CombineDiffCal-v1.rst`.